### PR TITLE
Code snippet for mapping approach

### PR DESCRIPTION
**Pull request by swalsh15**
Code snippet for console.log() to print() using regular expressions to parse and manually mapping. Only transforms console.log(), ignores all other functions in input file.

### CODE REVIEW

**Review by g-harel — APPROVED** [review]

File: portfolio/src/main/webapp/test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+*/
+
+let userCode = "console.log(x);\n"; 
+userCode += "   console.log(1 + 2);\n";
+userCode += " 1 * 5;\n"
+userCode += "console.log(\"simple string\");\n"
+userCode += " console.log(\"hello\" + \"world\")"; 
+
+// matches all console.log statements
+const consoleLogPattern = /console\.log\((.*)\)/g;
```
Comment (augerm) [inline portfolio/src/main/webapp/test.js:15]: Here's (definitely an edge case) that might cause problems here.

Very awkward for them to put this all on one line but it might cause problems.
`(() => { console.log("Hello World") })();`

Not against pushing this edge case to a future PR though - just add a comment for now.

File: portfolio/src/main/webapp/test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
```
Comment (augerm) [inline portfolio/src/main/webapp/test.js:1]: First off... Awesome start super exciting to see.
